trace-utils/src/trace_utils.rs

```diff
@@ -0,0 +1,274 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+use hyper::http::HeaderValue;
+use hyper::HeaderMap;
+use hyper::{body::Buf, Body, Client, Method, Request};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::{env, str};
+
+use prost::Message;
+
+use datadog_trace_protobuf::pb;
+
+const TRACE_INTAKE_URL: &str = "http://trace.agent.datadoghq.com/api/v0.2/traces";
```

Comment:
I think latest version is v0.7 ->https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/api/version.go#L120
Could we use latest endpoint?

Reply:
that version.go document outlines agent endpoint versions (for the tracers to use), rather than backend trace endpoints. the current trace_intake_url should be correct!